content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -148,6 +148,24 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
+The `Role` or `ClusterRole` a binding object refers to may not be modified.
```

Comment:
```suggestion
You cannot modify the `Role` or `ClusterRole` that a binding object refers to.
```
How about this wording?

Reply:
went with "You cannot modify which `Role` or `ClusterRole` a binding object refers to." instead